mewarpx/mewarpx/emission.py

```diff
@@ -741,28 +724,32 @@ def apply_Schottky_weights(self, particle_dict):
 
     def get_normals(self, x, y, z):
         """Calculate local surface normal at specified coordinates.
+
         Arguments:
             x (np.ndarray): x-coordinates of emitted particles (in meters).
             y (np.ndarray): y-coordinates of emitted particles (in meters).
             z (np.ndarray): z-coordinates of emitted particles (in meters).
+
         Returns:
             normals (np.ndarray): nx3 array containing the outward surface
                 normal vector at each particle location.
         """
         raise NotImplementedError('Normal calculations must be implemented by'
                                   + ' Emitter sub-classes.')
 
+
 class ZPlaneEmitter(Emitter):
 
     """This is the standard injection for a planar cathode."""
+
     geoms = ['Z', 'XZ', 'XYZ']
 
-    def __init__(self, conductor, T, xmin=None, xmax=None,
+    def __init__(self, conductor, T=None, xmin=None, xmax=None,
                  ymin=None, ymax=None, transverse_fac=1.0, **kwargs):
         """Initialize an emitter for a planar cathode.
 
         Arguments:
-            conductor (warp.Assembly): Conductor object, used to obtain work
+            conductor (warpx.Assembly): Conductor object, used to obtain work
```

Comment:
And rewrap line as needed please.
```suggestion
            conductor (:class:`mewarpx.assemblies.Assembly`): Conductor object, used to obtain work
```